### PR TITLE
Policy v1.18.4

### PR DESCRIPTION
**Pull request by mrmills93**
### What does this PR do?

Contains policy bug fixes found during windows beta agent QA

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Comment by lebauce** [comment]
Shouldn't we bump the version to 1.18.5 ?

File: runtime/default.policy [hunk 1]
```diff
@@ -378,7 +383,7 @@ rules:
       - os == "linux"
   - id: cryptominer_args
     description: A process launched with arguments associated with cryptominers
-    expression: exec.args_options in [~"cpu-priority*", ~"donate-level*"] || exec.args in [~"*stratum+tcp*"]
+    expression: exec.args_options in [~"cpu-priority*", ~"donate-level*"] || exec.args in [~"*stratum+tcp*", ~"*stratum+ssl*", ~"*stratum1+tcp*", ~"*stratum1+ssl*", ~"*stratum2+tcp*", ~"*stratum2+ssl*"]
```
Comment (paulcacheux) [inline runtime/default.policy:386]: let's keep the update to windows changes only

File: runtime/default.policy [hunk 2]
```diff
@@ -396,9 +401,9 @@ rules:
        exec.comm in HTTP_UTILS ||
        exec.file.path in SHELL_UTILS) &&
       process.parent.file.name in DATABASE_PROCESSES &&
-      !(process.ancestors.file.name == "initdb" &&
+      !(process.parent.file.name == "initdb" &&
```
Comment (paulcacheux) [inline runtime/default.policy:404]: same

File: runtime/default.policy [hunk 3]
```diff
@@ -956,7 +961,7 @@ rules:
       (process.parent.file.name in WEBAPP_PROCESSES || process.parent.file.name =~ "php*")
     filters:
       - os == "linux"
-  - id: procdump_usage
+  - id: procdump_execution
```
Comment (paulcacheux) [inline runtime/default.policy:964]: are those id changes in sync with backend rules ?

**Review by paulcacheux — CHANGES_REQUESTED** [review]

File: runtime/default.policy (hunk 3 above)
Comment (mrmills93) [reply runtime/default.policy:964]: yep!

Comment (mrmills93) [reply runtime/default.policy:964]: https://github.com/DataDog/security-monitoring/blob/main/workload-security/backend-rules/procdump_usage.yaml

**Pull request merged**